src/FirstSteps/RPG/Heroes/HeroesCreator.cs

```diff
@@ -28,5 +30,29 @@ public static Hero Create(string name, Races race)
                 return null;
             }
         }
+        public static Hero Create(Races race)
+        {
+
+            HeroModel heroModel = HeroesRepository.LoadHero();
+            
+            if (race == Races.Human)
```

Comment:
The `race` that you compare should be from the `heroModel`. It is saved in a JSON file and based on that you should know what kind of hero you have to create.